MaxText/elastic_train.py

```diff
@@ -187,7 +187,7 @@ def train_loop(config, elastic_manager, state=None):
   Args:
     config:
     state:
-    ckpt_path:
```

Comment:
why this change ?

Reply:
Looking at `main` you can see that the wrong name is listed in the argument list: https://github.com/AI-Hypercomputer/maxtext/blob/9cc5aec/MaxText/elastic_train.py#L190